### ISSUE TITLE
All old events displayed as "untitled"

### ISSUE THREAD

**Issue by RedShiba** [getsentry/self-hosted#1985]
### Self-Hosted Version

21.1.0

### CPU Architecture

x86_64

### Docker Version

20.10.16

### Docker Compose Version

1.28.5

### Steps to Reproduce

Hey. I want to create copy of my sentry production instance for some tests. I made a backups of clickhouse and postgress. Restored these backups on new host. Copied files into sentry-data volume and configs. Redis/Kafka was run from scratch. Also run all services using docker-compose. I see all events and issues in a databases. Also I see issues(old/new) on the UI. Everything looks good except old events on the UI. All old events can not be loaded and I see that as  "untitled". When I go inside issue to see the details I see infinite loading. It looks like 404 In logs "GET /api/0/issues/109471/events/latest/ HTTP/1.1 404" ("Event not found"). But I can see these events in clickhouse.
<img width="707" alt="paveikslas" src="https://user-images.githubusercontent.com/96052052/221024028-9aabd72e-bba4-496f-aece-f1e2df68c9a6.png">

If new events appear I can see new events in this case(no infinite loading anymore) but old events still look like "untitled"
<img width="605" alt="paveikslas" src="https://user-images.githubusercontent.com/96052052/221025085-fdd30ca5-ff98-4703-91f2-dcbad892d40c.png">

What am I missing, any thoughts?

### Expected Result

able to see old events after DB restoring and move configs to new VM

### Actual Result

can't see old events

**Comment by hubertdeng123** [comment]
I'm not quite sure exactly where that data might be since I'm not the most familiar with that code, but I took a glance into the Sentry codebase and found a few things.
`<untitled>` is set here:
https://github.com/getsentry/sentry/blob/master/src/sentry/eventtypes/base.py#L67

Event metadata seems to be coming from:
https://github.com/getsentry/sentry/blob/master/src/sentry/eventstore/models.py#L80

So this maybe has something to do with our node storage 🤔

**Comment by RedShiba** [comment]
nodestore_node table is empty in postgres even in production :/ 
I thought it was removed and migrated after update from 9.1.2 to 21.1.0. But I don't know where its stored now

I found here https://github.com/getsentry/sentry/blob/21.1.0/src/sentry/nodestore/README.rst
`Node Storage removes the size problem from these databases and stores the content in a traditional key/value database, which is more suitable for the size of Event.`
key/value db ..is it redis?

**Comment by hubertdeng123** [comment]
Glad to hear that @RedShiba . Would you mind posting your solution in case other people encounter the same issue?

**Comment by RedShiba** [comment]
sure, there is nothing special. When you said event metadata coming from nodestore.(there was my lack of knowledge), I checked table nodestore_node in postgres and found it almost empty. After that I checked pg_restore logs and found in the middle of restoring an error "No space left on device" when it tried to restore nodestore_node. 
Pg_restore output is huge and after the error it continued restoring other tables and finished "successfully". 
So I just cleared disk space up and everything restored well.
